tools/src/main/java/org/apache/kafka/tools/TopicCommand.java

```diff
@@ -210,7 +210,7 @@ private static Integer getReplicationFactor(TopicPartitionInfo tpi, PartitionRea
      */
     private static void ensureTopicExists(List<String> foundTopics, String requestedTopic, Boolean requireTopicExists) {
         // If no topic name was mentioned, do not need to throw exception.
-        if (!(requestedTopic.isEmpty() || !Optional.ofNullable(requestedTopic).isPresent()) && requireTopicExists && foundTopics.isEmpty()) {
+        if (requestedTopic != null && !requestedTopic.isEmpty() && requireTopicExists && foundTopics.isEmpty()) {
```

Comment:
it never pass null `requestedTopic`, and it results in `!Optional.ofNullable(requestedTopic).isPresent()` being unnecessary condition - it is always `false`

Reply:
Yeah, I know there is no null input case for `requestedTopic`. However, do we want to check `requestedTopic != null` to prevent null case in the future?

Reply:
that makes sense to me. Could you change the type from `String` to `Optional<String>` to handle the null?

Reply:
Updated it. Thanks.